x-pack/plugins/observability/public/pages/slo_outdated_definitions/index.tsx

```diff
@@ -0,0 +1,129 @@
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTablePagination, EuiText } from '@elastic/eui';
+import { useBreadcrumbs } from '@kbn/observability-shared-plugin/public';
+import { useKibana } from '../../utils/kibana_react';
+import { useLicense } from '../../hooks/use_license';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+import { useCapabilities } from '../../hooks/slo/use_capabilities';
+import { useFetchSloGlobalDiagnosis } from '../../hooks/slo/use_fetch_global_diagnosis';
+import { HeaderMenu } from '../overview/components/header_menu/header_menu';
+import { useFetchSloDefinitions } from '../../hooks/slo/use_fetch_slo_definitions';
+import { paths } from '../../../common/locators/paths';
+import { SloListEmpty } from '../slos/components/slo_list_empty';
+import { OutdatedSlo } from './outdated_slo';
+import { OutdatedSloSearchBar } from './outdated_slo_search_bar';
+
+export function SlosOutdatedDefinitions() {
+  const {
+    http: { basePath },
+  } = useKibana().services;
+  const { hasWriteCapabilities } = useCapabilities();
+  const { data: globalDiagnosis } = useFetchSloGlobalDiagnosis();
+  const { ObservabilityPageTemplate } = usePluginContext();
+
+  useBreadcrumbs([
+    {
+      href: basePath.prepend(paths.observability.slos),
+      text: i18n.translate('xpack.observability.breadcrumbs.slosLinkText', {
+        defaultMessage: 'SLOs',
+      }),
+      deepLinkId: 'observability-overview:slos',
+    },
+    {
+      text: i18n.translate('xpack.observability.breadcrumbs.slosOutdatedDefinitions', {
+        defaultMessage: 'Outdated SLO Definitions',
+      }),
+    },
+  ]);
+
+  const [search, setSearch] = useState<string>('');
+  const [activePage, setActivePage] = useState<number>(0);
+  const [perPage, setPerPage] = useState<number>(10);
+
+  const handlePerPageChange = (perPageNumber: number) => {
+    setPerPage(perPageNumber);
+    setActivePage(0);
+  };
+
+  const { hasAtLeast } = useLicense();
+
+  const { isLoading, data, refetch } = useFetchSloDefinitions({
+    name: search,
+    includeOutdatedOnly: true,
+    page: activePage + 1,
+    perPage,
+  });
+  const { total } = data ?? { total: 0 };
+
+  const hasRequiredWritePrivileges =
+    !!globalDiagnosis?.userPrivileges.write.has_all_requested && hasWriteCapabilities;
+
+  const hasSlosAndHasPermissions = hasAtLeast('platinum') === true && hasRequiredWritePrivileges;
+
+  return (
+    <ObservabilityPageTemplate
+      data-test-subj="slosOutdatedDefinitions"
+      pageHeader={{
+        pageTitle: i18n.translate('xpack.observability.slo.slosOutdatedDefinitions.pageTitle', {
+          defaultMessage: 'Outdated SLO Definitions',
+        }),
+      }}
+    >
+      <HeaderMenu />
+
+      {!hasSlosAndHasPermissions ? (
+        <EuiText>
+          {i18n.translate('xpack.observability.slo.slosOutdatedDefinitions.permissionsError', {
+            defaultMessage: 'You must have write permissions to access this page',
```

Comment:
This permission error doesn't seem specific enough. I think we should include the specific permissions needed.